test/web-platform-tests/to-run.yaml

```diff
@@ -103,6 +103,47 @@ window-screen-width.html: [fail, Test not applicable - no output device]
 
 ---
 
+DIR: custom-elements
+
+CustomElementRegistry.html: [fail, webidl2js doesn't deal well with tests using Proxies to verify properties access]
+Document-createElement.html: [fail, :defined is not defined and throws]
+HTMLElement-attachInternals.html: [fail, Not implemented]
+HTMLElement-constructor.html: [fail, webidl2js doesn't deal well with tests using Proxies to verify properties access]
+adopted-callback.html: [fail, "Failing test due to https://github.com/whatwg/dom/issues/813"]
+attribute-changed-callback.html: [fail, attributeChangedCallback doesn't work with CSSStyleDeclaration]
+custom-element-reaction-queue.html: [fail, Document.write implementation is not spec compliant]
+custom-element-registry/per-global.html: [fail, iframe location related issue]
+form-associated/ElementInternals-NotSupportedError.html: [fail, Not implemented]
+form-associated/ElementInternals-accessibility.html: [fail, Not implemented]
+form-associated/ElementInternals-labels.html: [fail, Not implemented]
+form-associated/ElementInternals-setFormValue.html: [timeout, Not implemented]
+form-associated/ElementInternals-validation.html: [fail, Not implemented]
+form-associated/form-associated-callback.html: [fail, Not implemented]
+form-associated/form-disabled-callback.html: [timeout, Not implemented]
+form-associated/form-reset-callback.html: [fail, Not implemented]
+htmlconstructor/newtarget.html: [fail, Currently impossible to get the active function associated realm]
+microtasks-and-constructors.html: [fail, Usage of external scripts doesn't block HTML parsing, https://github.com/jsdom/jsdom/issues/2413]
+parser/parser-constructs-custom-element-synchronously.html: [fail, Usage of external scripts doesn't block HTML parsing, https://github.com/jsdom/jsdom/issues/2413]
+parser/parser-fallsback-to-unknown-element.html: [fail, Usage of external scripts doesn't block HTML parsing, https://github.com/jsdom/jsdom/issues/2413]
+parser/parser-sets-attributes-and-children.html: [fail, Usage of external scripts doesn't block HTML parsing, https://github.com/jsdom/jsdom/issues/2413]
+parser/parser-uses-constructed-element.html: [fail, Usage of external scripts doesn't block HTML parsing, https://github.com/jsdom/jsdom/issues/2413]
+parser/serializing-html-fragments.html: [fail, parse5 doesn't support is attribute for serialization]
```

Comment:
Can we file an issue for this?

Reply:
I realized as I was writing the issue for this, that adding a new parse5 tree adapter method is probably overkill for handing the `is` attribute serialization. The parse5 API is already complex, I don't think adding more complexity into the mix is the right option. For this reason, I added some special logic to handle the `is` attribute in the `getAttrList` handler to add the `is` attribute if necessary there.